src/librustc_mir/const_eval.rs

```diff
@@ -666,63 +684,63 @@ pub fn const_eval_raw_provider<'tcx>(
         })
     }).map_err(|error| {
         let err = error_to_const_error(&ecx, error);
-        // errors in statics are always emitted as fatal errors
-        if tcx.is_static(def_id) {
-            // Ensure that if the above error was either `TooGeneric` or `Reported`
-            // an error must be reported.
+    // errors in statics are always emitted as fatal errors
```

Comment:
Why is this unindented?

Reply:
I was viewing all diffs in no-whitespace mode... thanks for catching this